MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -854,17 +867,39 @@ public override List<string> GetLogFilePaths(string basePath)
                         logFiles.Add($"{basePath}.fulltoc");
                     if (File.Exists($"{basePath}.log"))
                         logFiles.Add($"{basePath}.log");
-                    if (File.Exists($"{basePath}.skeleton"))
-                        logFiles.Add($"{basePath}.skeleton");
                     if (File.Exists($"{basePath}.state"))
                         logFiles.Add($"{basePath}.state");
                     if (File.Exists($"{basePath}.subcode"))
                         logFiles.Add($"{basePath}.subcode");
                     if (File.Exists($"{basePath}.toc"))
                         logFiles.Add($"{basePath}.toc");
+
+                    // Include .hash and .skeleton for all files in cuesheet
+                    CueSheet? cueSheet = new SabreTools.Serialization.Files.CueSheet().Deserialize($"{basePath}.cue");
```

Comment:
You can use `var` here if you want. The righthand side makes it apparent what the type is.